core/Dispatch.hpp

```diff
@@ -98,6 +99,7 @@ namespace olympia
 
         const uint32_t num_to_dispatch_;
         uint32_t credits_rob_ = 0;
+        uint32_t dispatch_queue_depth_;
```

Comment:
Needs to be `const`

Reply:
Resolved in [6b0e363](https://github.com/riscv-software-src/riscv-perf-model/pull/140/commits/6b0e3633c1b7331170afda8920f596f9fd16b233)